src/components/react-hook-form/utils/cancel-button.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Button } from '@mui/material';
+import { FormattedMessage } from 'react-intl';
+
+const styles = {
+    removeButtonTextColor: (theme) => ({
+        color: theme.palette.text.primary,
+    }),
+};
+const CancelButton = ({
+    onClick,
+    variant,
+    disabled,
+    withCustomColor = true,
+}) => {
+    return (
+        <Button
+            onClick={onClick}
+            variant={variant}
+            disabled={disabled}
+            sx={withCustomColor ? styles.removeButtonTextColor : null}
```

Comment:
Why you use `sx` and not the `color` prop ?
https://mui.com/material-ui/api/button/#Button-prop-color